### DISCUSSION TITLE
Mesh problem

### DISCUSSION THREAD

**Discussion by byeonghoon-ahn** [mdolab/dafoam#210] [General]
Hello Dr He,
I used openfoam's snappyHex Mesh in DAFoam to draw the grid. There is no problem checking the grid in Fig1.

![image](https://user-images.githubusercontent.com/77006538/141707959-c9107373-20fe-4dc0-9db6-1432e5fa6475.png)
Fig1. After drawing snappyHex grid, using checkMesh check the grid

However, there is a problem with the grid during dafoam calculation in Fig2. 
![image](https://user-images.githubusercontent.com/77006538/141708190-c190cb30-d60a-43ad-8ff8-e639776a0220.png)
Fig 2

How can i solve this problem?

**Comment by friedenhe** [comment]
It is possible that your FFD does not completely contain the design surface geometry. Check your FFD and mesh in paraview.

**Reply by byeonghoon-ahn** [reply]
I did fix this problem after modifying FFD, Thank you for your solution!
